### DISCUSSION TITLE
sorting a List of related foreign-key records?

### DISCUSSION THREAD

**Discussion by davidroberthoare** [vitalik/django-ninja#654] [Q&A]
My Schema are like this:

```
class TagSchema(ModelSchema):
    class Config:
        model = Tag
        model_fields = '__all__'

class EventTypeSchema(ModelSchema):
    class Config:
        model = EventType
        model_fields = '__all__'

class PartSchema(ModelSchema):
    tags: List[TagSchema]
    duration: int = Field(None, alias="duration")
    class Config:
        model = Part
        model_fields = '__all__'

class EventSchema(Schema):
    id: int
    channel_id: int
    type: EventTypeSchema
    name: str=None 
    start_date: date
    details: str=None
    parts: List[PartSchema]
```

And I can successfully now list all events, with their associated parts and tags, etc...
```
events = Event.objects.filter(channel_id=chan).order_by("start_date", "name")
```

However, the related lists ('parts' and 'tags') are not ordered. For example, I would like to order the parts list for each event by the "part.start_time" field. But if I include `order_by="parts__start_time`  it just duplicates the Event row in the result (and doesn't sort the parts either)...
How can I accesss/sort the foreign-key-related lists that are being returned so nicely in this schema? :)
Thanks!
David

**Comment by paqstd-dev** [comment]
Hello. For models, you can specify a default sort:

``` python
# app/models.py
class Part(models.Model):
    ...
    class Meta:
        ordering = ('start_time',)
